src/pages/index.tsx

```diff
@@ -107,7 +107,7 @@ const Index: FC = () => {
       setFilteredTask(taskMap);
     }
 
-    return(() => {
+    return (() => {
       setFilteredTask([]);
     });
   }, [isLoading, response]);
```

Comment:
This file should not change at all as I can see there are only lint fix clears here it does not have any relation to your feature.

Reply:
i know , i don' think it would cause any issues

Reply:
it will in someone else's.

Reply:
got it

Reply:
fixed